### ISSUE TITLE
NodeJS util.promisify issues

### ISSUE THREAD

**Issue by liljosi** [conekta/conekta-node#63]
Trying to make Conekta methods work with promises, but i keep getting a pending promise and no logs are caught in Conekta account concerning HTTP requests. Any advice? or work arounds? Another error i get is "cannot read property post of undefined". It would be great if you guys could help.
![screen shot 2018-08-09 at 9 31 57 am](https://user-images.githubusercontent.com/28731155/43905857-8f724b8e-9bb7-11e8-9157-149b14502c6e.png)
![screen shot 2018-08-09 at 9 32 42 am](https://user-images.githubusercontent.com/28731155/43905882-980872b4-9bb7-11e8-85d5-4625448c823d.png)

**Comment by cesargdm** [comment]
I made a wrapper like this:

```
const createClient = ({ name, email, phone, paymentSources = [] }) => {
  return new Promise((resolve, reject) => {
    conekta.Customer.create(
      {
        name,
        email,
        phone,
        payment_sources: paymentSources,
      },
      (err, res) => (err ? reject(err) : resolve(res))
    )
  })
}
```

**Comment by andidev** [comment]
I have the same issue. I have not tried yet  ut I think this should work.

```javascript 
conekta.Customer.createWithPromise = util.promisify(conekta.Customer.create);
```

**Comment by KarlaRobinson** [comment]
We are working on a version that is compatible with promises. It should be released in a few days.

**Comment by KarlaRobinson** [comment]
Thanks for everyones suggestions. We have released a new version 3.5.2 which is compatible with promises.
